Week1/homework/worldQueries.js

```diff
@@ -0,0 +1,64 @@
+const mysql = require('mysql');
+
+// Create the connection
+const connection = mysql.createConnection({
+  host     : 'localhost',
+  user     : 'hyfuser',
+  password : 'hyfpassword',
+  database : 'world',
+  // port : 3307
+});
+connection.connect();
+
+
+// write function to avoid repetition in code
+function getResults(sql) {
+  connection.query(sql, (err, results) => {
+    if (err) {
+      return console.error(err.message);
+    }
+   console.log(results)})
+}
+
+// What are the names of countries with population greater than 8 million?
+const sql1 = 'SELECT Name FROM country WHERE Population > 8000000'
```

Comment:
nitpick: why not semicolon at the end ?
Please be consistent with the usage of semi-colons.
I see it used for some statements but not for all

Reply:
Thank you for your feedback. I fixed the files and pushed my work again with modifications. I will ensure that will be considered in the future.